cmd/commands/cli/command.go

```diff
@@ -356,6 +357,53 @@ func (c *cliApp) unlock(argsString string) {
 	success(fmt.Sprintf("Identity %s unlocked.", identity))
 }
 
+func (c *cliApp) payout(argsString string) {
+	const usage = "payout command:\n    set"
+	if len(argsString) == 0 {
+		info(usage)
+		return
+	}
+
+	args := strings.Fields(argsString)
+	if len(args) < 1 {
+		info(usage)
+		return
+	}
+
+	action := args[0]
+	switch action {
+	case "set": // Known sub-commands.
+	default:
+		warnf("Unknown sub-command '%s'\n", action)
+		fmt.Println(usage)
+		return
+	}
+
+	if action == "set" {
+		payoutSignature := "payout set <identity> <ethAddress>"
+		if len(args) < 2 {
+			info("Press tab to select identity.\n", payoutSignature)
+			return
+		}
+
+		var identity, ethAddress string
+		if len(args) == 3 {
```

Comment:
There are 4 checks:
`if len(argsString) == 0 {`
`if len(args) < 1 {`
`if len(args) < 2 {`
`if len(args) == 3 {`

But looks like only one requried `if len(args) != 3 { return error }`

Reply:
You are right - only one is requried `if len(args) == 3 { ... }` to check if command, sub-command and parameter structure is valid.
All other checks are for console to help user to navigate through commands and parameters.

Reply:
Reduced checks